frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -1,24 +1,28 @@
 import { interpolateYlOrRd } from "d3-scale-chromatic";
 import {
+  DatasetComponentOption,
   DefaultLabelFormatterCallbackParams,
   EChartsOption,
   ScatterSeriesOption,
 } from "echarts";
 import { State } from "../../common/store";
-import { CellTypeSummary, Tissue } from "../../common/types";
-import ReplaySVG from "./icons/replay.svg";
+import {
+  CellTypeSummary,
+  GeneExpressionSummary,
+  Tissue,
+} from "../../common/types";
 
 export const MAX_FIRST_PART_LENGTH_PX = 16;
-export const X_AXIS_CHART_HEIGHT = "200px";
-export const Y_AXIS_CHART_WIDTH = "300px";
+export const X_AXIS_CHART_HEIGHT_PX = 80;
+export const Y_AXIS_CHART_WIDTH_PX = 300;
 
-const REPLAY_SVG_WIDTH_PX = 12;
+const Y_AXIS_BOTTOM_PADDING = "10px";
 
-interface CreateOptionsProps {
-  geneNames: string[];
-  cellTypeNames: string[];
-  commonOptions: EChartsOption;
-}
+const COMMON_OPTIONS = {
```

Comment:
since `chartData` is now only used by `Chart` component, `commonOptions` that was previously shared among all three charts, which had `chartData` can now be declared as a constant!